src/main/scala/org/kframework/kore/parser/KoreToText.scala

```diff
@@ -42,8 +42,25 @@ object KoreToText {
       " " +
       apply(att)
 
+    case HookSortDeclaration(params, sort, att) =>
+      "hook-sort" +
+        " " +
+        apply(sort) +
+        " " +
+        apply(att)
+
     case SymbolDeclaration(symbol, argSorts, returnSort, att) =>
       "symbol " +
+        apply(symbol) +
```

Comment:
I'm not sure about the default Scala coding conventions, but here you're using a different indentation from below (i.e. for the string concatenation in HookSymbolDeclaration)

Reply:
Thanks. Fixed.